external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MariaDBKrbIntegrationSuite.scala

```diff
@@ -25,9 +25,9 @@ import org.apache.spark.sql.execution.datasources.jdbc.connection.SecureConnecti
 import org.apache.spark.tags.DockerTest
 
 /**
- * To run this test suite for a specific version (e.g., mariadb:10.5.12):
+ * To run this test suite for a specific version (e.g., mariadb:10.5.13):
  * {{{
- *   ENABLE_DOCKER_INTEGRATION_TESTS=1 MARIADB_DOCKER_IMAGE_NAME=mariadb:10.5.12
+ *   ENABLE_DOCKER_INTEGRATION_TESTS=1 MARIADB_DOCKER_IMAGE_NAME=mariadb:10.5.13
```

Comment:
As far as I could tell the referenced image no longer exists in docker hub.

Reply:
cc @sarutak

Reply:
`10.5.12` is still available.
https://hub.docker.com/layers/mariadb/library/mariadb/10.5.12/images/sha256-a5ebad8f03ea9352ccf1ec3c84d91c2112ba86bbface1b57091f37aa7899d6c9?context=explore

But it's OK to upgrade.

Reply:
@tdg5 If you'd like to change here, please change [this part](https://github.com/apache/spark/pull/34745/files#diff-cb12fa58307c2155fe2156d347d7a7917f1c755c34488a741dfb06653f83f364L41) too.

Reply:
I'm going to roll this back since it is not required; sorry for the confusion.